gpii/node_modules/flowManager/test/UpdateTests.js

```diff
@@ -84,92 +78,95 @@ gpii.tests.flowManager.update.testSecondUpdateRequest = function (data) {
 
 gpii.tests.flowManager.update.clearActiveSessions = function (activeSessions) {
```

Comment:
The function ```gpii.tests.flowManager.update.testFirstUpdateRequest()``` at line 71 and the function ```gpii.tests.flowManager.update.testSecondUpdateRequest``` at line 75 are exactly same. One of them can be removed.